cli/tests/environment_sync.sh

```
@@ -0,0 +1,64 @@
+echo "Testing environment sync command..."
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)